### PR TITLE
feat/storybook-7-upgrade-13

### PR DESCRIPTION
**Pull request by panvourtsis**

### CODE REVIEW

File: src/components/Modal/Modal.tsx [hunk 1]
```diff
@@ -22,7 +23,7 @@ export type ModalProps = {
   isContentPadded?: boolean;
 };
 
-const Modal: React.FCC<ModalProps> = ({
+const Modal: React.FC<React.PropsWithChildren<ModalProps>> = ({
```
Comment (kostasdano) [inline src/components/Modal/Modal.tsx:26]: this could probably be FCC

File: src/components/List/ListSection.tsx [hunk 2]
```diff
@@ -13,7 +14,9 @@ export type ListSectionProps = {
   rowSize?: ListRowSize;
 } & SectionProps<unknown>;
 
-const ListSection: React.FCC<ListSectionProps> = (props) => <AriaSection {...props} />;
+const ListSection: React.FC<React.PropsWithChildren<ListSectionProps>> = (props) => (
```
Comment (kostasdano) [inline src/components/List/ListSection.tsx:17]: this could probably be FCC

File: src/components/List/ListItem.tsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ export type ListItemProps = {
   /** @default normal */
   rowSize?: ListRowSize;
 };
-const ListItem: React.FCC<ListItemProps> = (props) => (
+const ListItem: React.FC<React.PropsWithChildren<ListItemProps>> = (props) => (
```
Comment (kostasdano) [inline src/components/List/ListItem.tsx:14]: this could probably be FCC

**Review by kostasdano — APPROVED** [review]

File: src/components/Modal/Modal.tsx (hunk 1 above)
Comment (panvourtsis) [reply src/components/Modal/Modal.tsx]: fixed here https://github.com/Orfium/orfium-ictinus/pull/720/commits/a4a6b5364393adb1f2351a633b714e8d7d8eebdc

File: src/components/List/ListItem.tsx (hunk 3 above)
Comment (panvourtsis) [reply src/components/List/ListItem.tsx]: resolved here https://github.com/Orfium/orfium-ictinus/pull/720/commits/f50892c7c98ff1d524fd9d7c6393c46fc9782161

File: src/components/List/ListSection.tsx (hunk 2 above)
Comment (panvourtsis) [reply src/components/List/ListSection.tsx]: resolved here https://github.com/Orfium/orfium-ictinus/pull/720/commits/f50892c7c98ff1d524fd9d7c6393c46fc9782161

**Pull request merged**